playbook/app/pb_kits/playbook/pb_dialog/index.js

```diff
@@ -0,0 +1,75 @@
+import PbEnhancedElement from "../pb_enhanced_element";
+
+const DIALOG_WRAPPER_SELECTOR = "[data-pb-dialog-wrapper]";
+
+export default class PbDialog extends PbEnhancedElement {
+  static get selector() {
+    return DIALOG_WRAPPER_SELECTOR;
+  }
+
+  connect() {
+    window.addEventListener("DOMContentLoaded", () => this.setupDialog())
+    window.addEventListener("turbo:frame-load", () => this.setupDialog())
```

Comment:
Will this change with the turbo frames events just work with the places in Nitro that are using the dialog within turbo frames? Or will we need to make changes in Nitro to account for it? We want to make sure to test locations in Nitro that are using dialogs with Turbo

Reply:
@nidaqg I just saw this now. Sorry. But yes, this is the same code we had before, so everything should stay the same. We added some Nitro places using Turbo, and Ninja tested it.